CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.3.0 (2019-05-07)
+
+- Added methods to `SkyAuthTokenProvider` for getting a decoded token.  This allows consumers to access properties like the user's current zone in order to make a request to the appropriate zone endpoint. [#17](https://github.com/blackbaud/skyux-http/pull/17)
```

Comment:
I'd tweak this to remove articles and make the second sentence a fragment as per our usual style. Does this work?

`Added methods to `SkyAuthTokenProvider` to get decoded token. Enables consumers to access properties such as user's current zone to make requests to appropriate zone endpoint.`

Reply:
I tweaked some of the verbiage.  Removing all the articles made it sound nonsensical/caveman-like, so I kept most of them in.